docs/en/development/build-cross-riscv.md

```diff
@@ -1,10 +1,10 @@
 ---
+slug: /en/development/build-cross-riscv
```

Comment:
Apparently this is validated by the doc check and now doc check fails, and does not allow to run other checks.

Reply:
- #40733